moto/rds/utils.py

```diff
@@ -37,6 +37,7 @@ class DbInstanceEngine(str, Enum):
     CUSTOM_SQLSERVER_WEB = "custom-sqlserver-web"
     MARIADB = "mariadb"
     MYSQL = "mysql"
+    NEPTUNE = "neptune"  # TODO: Only allow from Neptune client?
```

Comment:
No need.  The backend for RDS and Neptune is one and the same (as it is with real AWS).  Only boto3 client-side validation prevents sending `neptune` as the engine type (when using the `rds` client).